include/z64bgcheck.h

```diff
@@ -52,10 +52,26 @@ typedef struct {
 } CollisionPoly; // size = 0x10
 
 typedef struct {
-    /* 0x00 */ u16 cameraSType;
-    /* 0x02 */ s16 numCameras;
-    /* 0x04 */ Vec3s* camPosData;
-} CamData;
+    /* 0x0 */ u16 setting; // camera setting described by CameraSettingType enum
+    /* 0x2 */ s16 count; // only used when `bgCamData` is a list of points used for crawlspaces
+    /* 0x4 */ s16* bgCamData; // s16 data grouped in threes (ex. Vec3s), is usually of type `BgCamData`, but can be a list of points of type `Vec3s` for crawlspaces
```

Comment:
just to put this somewhere other than lost on discord
```suggestion
    /* 0x4 */ s16* camFuncData; // s16 data grouped in threes (ex. Vec3s), is usually of type `BgCamData`, but can be a list of points of type `Vec3s` for crawlspaces
```
\+ idk about how to add onto the comment without writing a whole essay